Makefile

```diff
@@ -336,6 +338,11 @@ kind-ha-single-zone-up: $(KIND) $(KUBECTL) $(HELM) $(YQ)
 kind-ha-single-zone-down: $(KIND)
 	./hack/kind-down.sh --cluster-name gardener-local-ha-single-zone --path-kubeconfig $(REPO_ROOT)/example/provider-local/seed-kind-ha-single-zone/base/kubeconfig
 
+kind2-ha-single-zone-up: $(KIND) $(KUBECTL) $(HELM) $(YQ)
```

Comment:
It feels like there are now lots of make targets with very similar, marginally different actions. What do you think about unifying some of them?

Reply:
Yeah, that bothered me a bit as well, but I couldn't figure out a good way to do it, so I'm open to suggestions. One benefit of having the multiple targets is that you wouldn't have to type in any extra parameters. However, it makes it harder working on this `Makefile`. Perhaps we should also think about splitting it in separate ones?

Reply:
I was thinking about using more variables similar to `IPFAMILY` and `KUBECONFIG` and then using those, but that is probably best handled in a separate PR. When this is merged I will try it.